### PR TITLE
Vagrant all in one

### LINKED ISSUE #175: ➹ Add a vagrant file with an all-in-one VM solution to easy the first use
**Is your feature request related to a problem? Please describe.**
As a first time secureCodeBox user it would be great to start easily with an all-in-one VM box to start with the secureCodeBox Project. 

**Describe the solution you'd like**
Add a vagrant file including the complete stack fully installed and ready to use - with batteries included.
- minikube + kubectl
- complete SCBv2 stack installed
- demo apps + scans + hooks installed

Additional add a vagrant bootstrap documentation and describe how to start with the vagrant setup.

**Describe alternatives you've considered**

**Additional context**

The secureCodeBox Version 1 already contains an vagrant file (but with an docker-compose bootstrapping instead of kubernetes): https://github.com/secureCodeBox/secureCodeBox/blob/v1.2/Vagrantfile

** Documentation **

- [ ] Add a lightweight vagrant how to install documentation section to the installation section

### PR DESCRIPTION
**Pull request by Weltraumschaf**
Adds a Vagrant base for #175. 

This needs the install script which is WIP at themoment.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#
+# All in one Vagrant box.
```
Comment (SebieF) [inline Vagrantfile:2]: Maybe complement: "All in one Vagrant box for the secureCodeBox"

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  end
+
+  config.vm.provider :libvirt do |c|
+    # https://github.com/vagrant-libvirt/vagrant-libvirt
+    c.memory = memory
+    c.cpus = cpus
+  end
+
+  config.vm.provision :shell, path: "#{base_dir}/bin/install-minikube.sh"
+  # Using sudo -g to run the command w/ newly created group from isntallation w/o the need of relogin.
```
Comment (SebieF) [inline Vagrantfile:42]: Typo: installation

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  # Redirecting STDERR to /dev/null because these morons print download progress
```
Comment (SebieF) [inline Vagrantfile:43]: Insulting? :D

File: Vagrantfile [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  # for the images to STDERR which clutters up the Vagrant output w/ error output!
+  config.vm.provision :shell, privileged: false, inline: "sudo -g docker minikube start 2>/dev/null"
+
+  # Do not automaticall install VirtualBox guest additions, if available.
```
Comment (SebieF) [inline Vagrantfile:47]: Typo: automatically

**Review by SebieF — APPROVED** [review]

File: Vagrantfile (hunk 3 above)
Comment (Weltraumschaf) [reply Vagrantfile:43]: Jep, they deserve it?

**Review by SebieF — APPROVED** [review]

**Pull request merged**